float32 d_amt
---
bool success